<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo State Machine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="DisplayDemoStateMachineImageWidget" name="visualStateMachineWidget" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Control</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="startStatemachineButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Events</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonPressButton">
         <property name="text">
          <string>Button Press</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mailSensorMsg">
         <property name="text">
          <string>Mail Sensor Msg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="batteryTimerExpiredButton">
         <property name="text">
          <string>Battery Timer Expired</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="radioFatalErrorButton">
         <property name="text">
          <string>Radio Fatal Error</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DisplayDemoStateMachineImageWidget</class>
   <extends>QWidget</extends>
   <header>displaydemostatemachineimagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
